css/css-break/block-fragmentation-001-print.html

```diff
@@ -5,7 +5,7 @@
   but does get printed on the next page.
 </title>
 <meta name="flags" content="may print">
```

Comment:
Does "may print" mean anything? Not found anywhere else.
And the ref has a class named "reftest-paged". Also not found anywhere else.
How does this work?

Reply:
The `-print` in the filename is the special bit, see https://web-platform-tests.org/writing-tests/print-reftests.html. The "print" flag indeed isn't used anywhere else and it predates https://github.com/web-platform-tests/wpt/commit/a8b3dbf304ebc9f52fccdbcd1c22c139010efef9 so I'll remove it.